pom.xml

```diff
@@ -0,0 +1,208 @@
+        <version>2.19.1</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-surefire-provider</artifactId>
+            <version>1.0.1</version>
+          </dependency>
+        </dependencies>
+        <configuration>
+          <groups>Mock</groups>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <version>4.9.10</version>
+        <executions>
+          <execution>
+            <id>get-the-git-infos</id>
+            <goals>
+              <goal>revision</goal>
+            </goals>
+            <phase>initialize</phase>
+          </execution>
+        </executions>
+        <configuration>
+          <generateGitPropertiesFile>true</generateGitPropertiesFile>
+          <generateGitPropertiesFilename>${project.build.outputDirectory}/git.properties
+          </generateGitPropertiesFilename>
+          <includeOnlyProperties>
+            <includeOnlyProperty>^git.build.(time|version)$</includeOnlyProperty>
+            <includeOnlyProperty>^git.branch$</includeOnlyProperty>
+            <includeOnlyProperty>^git.commit.(id.full|time)$</includeOnlyProperty>
+            <includeOnlyProperty>^git.dirty$</includeOnlyProperty>
+          </includeOnlyProperties>
+          <commitIdGenerationMode>full</commitIdGenerationMode>
+          <failOnUnableToExtractRepoInfo>false</failOnUnableToExtractRepoInfo>
+          <failOnNoGitDirectory>false</failOnNoGitDirectory>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+  <dependencies>
+    <!--system scope-->
+    <dependency>
+      <groupId>com.avispl.symphony.api</groupId>
+      <artifactId>symphony-api-commons</artifactId>
+      <version>5.8.0</version>
+      <scope>system</scope>
+      <systemPath>${basedir}/lib/symphony-api-commons-5.8.0.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>com.avispl.symphony.api</groupId>
+      <artifactId>symphony-api-dal</artifactId>
+      <version>5.8.0</version>
+      <scope>system</scope>
+      <systemPath>${basedir}/lib/symphony-api-dal-5.8.0.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>com.avispl.symphony.dal</groupId>
+      <artifactId>symphony-dal-commons</artifactId>
+      <version>5.9.0</version>
+      <scope>system</scope>
+      <systemPath>${basedir}/lib/symphony-dal-commons-5.9.0.jar</systemPath>
+    </dependency>
+    <dependency>
+      <groupId>org.icmp4j</groupId>
+      <artifactId>icmp4j</artifactId>
+      <scope>system</scope>
+      <systemPath>${basedir}/lib/icmp4j-1018.jar</systemPath>
+      <version>1018</version>
+    </dependency>
+
+    <!--dependencies of symphony-dal-commons-->
+    <dependency>
+      <groupId>commons-beanutils</groupId>
+      <artifactId>commons-beanutils</artifactId>
+      <version>1.9.4</version>
```

Comment:
We'll need to make sure all dependencies are up to date with core now, so we trigger dependabot alerts as little as possible